### ISSUE TITLE
Development: please branch the 3.0 release

### ISSUE THREAD

**Issue by leamas** [Rasbats/shipdriver_pi#337]
There is already patches piling up after releasing 3.0, and there will probable be more. We now need to keep track of bugfix-only patches while heading for next 3.0.x release in a not that distant future.

OTOH, there is also work to do which includes bigger changes like library handling and #331. These are bigger changes which eventually will end up in something like 3.1 or 4.0.

To handle this  I suggest that we adhere to the standard workflow and branches a v3.0 branch from current master. This means that bug-fix-only patches will go to the v3.0 branch while bigger changes goes to master only.

Of course, bugfixes needs to be applied to both branches, especially in the beginning before they diverge. This is done using `git cherry-pick`, please let me know if there are questions. 

TL;DR: please do:

     $ git checkout -b v3.0 master
     $ git push origin v3.0:v3.0

**Comment by Rasbats** [comment]
How does this impact on update-templates?

git cherry-pick is another new experience.

**Comment by Rasbats** [comment]
v3.0 now in place.

**Comment by leamas** [comment]
> v3.0 now in place.

Great!

> How does this impact on update-templates?

Not at all. update-templates always lives in the master branch, and is auto-updated when used.

> git cherry-pick is another new experience.

OK. I'll try to show how it works as soon as there is a need i. e., when we have a bugfix patch in v3.0 which we need to have also in master. It's like merging, but for a single commit rather than a branch.

**Comment by leamas** [comment]
The issue as such is fixed. Thanks! Closing.
